data/base-spells.yaml

```diff
@@ -149,81 +149,81 @@ khri_preps:
 
 barb_abilities:
   Python:
-    activate: form python
-    start: imitating the cunning ways of the python
-    expire: finish practicing the Form of the Python
+    start_cmd: form python
```

Comment:
Maybe I'm weird, but I prefer to spell out abbreviations in configuration like this. I feel like it makes it easier on the next dev who is likely unfamiliar with the code. Thoughts?